stratified-split/script.whizzml

```diff
@@ -0,0 +1,82 @@
+;; Transposes the matrix: [[a b] [c d]] to [[a c] [b d]]
+
+(define (_transpose m)
+  (apply map list m))
+
+;; Creates the each dataset by sampling and using out-of-bag
```

Comment:
typo (the).  also, why the blank lines between the comment of a function and its definition?